### PR TITLE
editorial review of bridge docs

### PR DESCRIPTION
**Pull request by adamralph**
relates to https://github.com/Particular/docs.particular.net/pull/5705

### CODE REVIEW

**Comment by adamralph** [comment]
recent activity

**Comment by adamralph** [comment]
activity

File: nservicebus/bridge/performance.md [hunk 1]
```diff
@@ -1,48 +1,47 @@
...
 ```mermaid
 graph LR
-  Br2[Bridge 2<br /><sup>Endpoint A<br />Endpoint D</sup>]
+  Br2[Bridge 2]
   A[Endpoint A] --- Br2
   B[Endpoint B]
   Br2 --- D[Endpoint D]
-  Br1[Bridge 1<br /><sup>Endpoint B<br />Endpoint C</sup>] --- C[Endpoint C]
+  Br1[Bridge 1] --- C[Endpoint C]
   B --- Br1
```
Comment (adamralph) [inline nservicebus/bridge/performance.md:45]: ## Before

image

The extra annotations in the bridge boxes really don't add anything, and they're quite ugly.

## After

image

**Comment by adamralph** [comment]
recent activity

**Comment by adamralph** [comment]
recent activity

**Review by SzymonPobiega — APPROVED** [review]

**Review by dvdstelt — APPROVED** [review]

**Pull request merged**